index.bs

```diff
@@ -647,15 +650,19 @@ An {{XRReferenceSpace}} is obtained by calling {{XRSession/requestReferenceSpace
 
   - Passing a {{XRReferenceSpaceOptions/type}} of <dfn enum-value for="XRReferenceSpaceType">unbounded</dfn> creates an {{XRUnboundedReferenceSpace}} instance if supported by the [=XR device=] and the {{XRSession}}.
 
-The <dfn for="XRReferenceSpace">session</dfn> attribute is the {{XRSession}} that created the {{XRReferenceSpace}}.
+Each {{XRReferenceSpace}} has a 
```

Comment:
Was this sentence supposed to have more in it?

Reply:
I... honestly have no idea what this was intended to say. 😅 Removed.